src/cli/repl.rs

```diff
@@ -263,11 +270,12 @@ impl Repl<F> {
                             zstore: zstore.unwrap(),
                         };
 
-                        lurk_proof.persist(proof_id)?;
-                        lurk_proof_meta.persist(proof_id)?;
-                        commitment.persist(proof_id)?;
+                        lurk_proof.persist_at(proof_path)?;
+                        lurk_proof_meta.persist(proof_key)?;
+                        claim_comm.persist(claim_hash)?;
```

Comment:
This is weird, and makes me wonder whether the `claim_hash` was indeed calculated as intended. You're storing a commitment but using the *hash* of the hidden value as part of the key used to name it? It's very hard to imagine a set of circumstances where this is actually what you want. Maybe I'm missing something.